### PR TITLE
Nav bar Fixed 

### LINKED ISSUE #199: Fix Navigation Of Section
## Issue Type:
[ x] Bug Report 
[  ]  Feature Request 
[  ] Documentaion

## **Describe the bug**

When we get to each section then we got heading hidden under the navigation bar

## **Possible solution**

We have  to reduce the size of navigation bar not directly because if we reduce directly the switch button gets misplaced

### PR DESCRIPTION
**Pull request by rohank2502**
# Description

The issue of Navbar is fixed 

Fixes #199 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Explain the Testing instructions

**Test Configuration**: 
* Operating system:WIN
* Version:10
* Text-editors used:VSCODE
 
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
# ATTACH SCREEN-SHOTS

https://rohank2502.github.io/SFC-foundations/

### CODE REVIEW

**Comment by rohank2502** [comment]
@himanshu007-creator  Fixed that issue just we have to ensure that every new page must have a height of 10vmin in the header class.

File: css/style.css [hunk 1]
```diff
@@ -306,6 +307,32 @@ header #menu-bar{
     box-shadow: 0 1rem 1rem rgba(0,0,0,.2);
 }
 
+.banner{
+    margin-top: 3vmin;
+    width: 70%;
+}
+
+
```
Comment (1502shivam-singh) [inline css/style.css:315]: Remove these line gaps. 1 line break here is enough just like above ones

File: css/Join-Us/joinus.css [hunk 2]
```diff
@@ -491,9 +492,9 @@ header button .icon-box .fa{
 
 
 
-.dark-mode .hashheading{
+.dark-mode .subheading{
     color: white;
-    font-size: 6vmin;
+    /* font-size: 6vmin; */
```
Comment (1502shivam-singh) [inline css/Join-Us/joinus.css:497]: Remove this comment here, if no longer needed.

**Comment by himanshu007-creator** [comment]
@1502shivam-singh  nice work, @rohank2502 please make the changes

**Comment by rohank2502** [comment]
> @1502shivam-singh nice work, @rohank2502 please make the changes

Ok thanks will do

**Comment by rohank2502** [comment]
Done Changes @himanshu007-creator

**Review by himanshu007-creator — APPROVED** [review]
LGTM

**Pull request merged**